### PR TITLE
autoconf: Add autoconf 2.72

### PR DESCRIPTION
**Pull request by lazka**
All the removed patches (compared to 2.71) are included in this new release

### CODE REVIEW

File: autoconf2.72/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  # license exception
+  install -Dm644 ${_realname}-${pkgver}/COPYING.EXCEPTION \
+    $pkgdir/usr/share/licenses/autoconf${pkgver}/COPYING.EXCEPTION
+
+  find ${pkgdir}/usr/share
+
+  # conflict with bintuils
+  rm -f ${pkgdir}/usr/share/info/standards.info
+  # conflicts with autoconf
+  rm -rf ${pkgdir}/usr/share/{emacs,info}
```
Comment (MehdiChinoune) [inline autoconf2.72/PKGBUILD:65]: Those lines are not needed
binutils doesn't have that file. and there is no package named autoconf.
Beside that they are suffixed there is no way they conflict with any other package.

Comment (lazka) [reply autoconf2.72/PKGBUILD:65]: we have three other autoconf packages. Do you mean we should have this files only in the newest version?

Comment (MehdiChinoune) [reply autoconf2.72/PKGBUILD:65]: Sorry, I was talking about man files which were not removed.
We could move those files for the latest autoconf. Yes that's my idea. No other package have them so why remove them.

Comment (lazka) [reply autoconf2.72/PKGBUILD]: fair enough, done

**Comment by jannick0** [comment]
... and function CHECK runs through smoothly.

**Comment by MehdiChinoune** [comment]
> ... and function CHECK runs through cleanly (without GO and FORTRAN being installed).

Are you asking or reporting?

**Comment by jannick0** [comment]
> > ... and function CHECK runs through cleanly (without GO and FORTRAN being installed).
> 
> Are you asking or reporting?

That was an assertion, of course. :) 

A more general remark: What would be great is to make sure - for fundamental packages like `autoconf` as a minimum - that the package test suite runs through cleanly or with well-known failures not important on the platform.  

Updating packages is important, of course, which might be automatized somehow.  However a clean build is only the first step for generating a working package, but does not reduce the risk of bugs or failures as running the package test suite if available and monitoring the test results. 

See for a package not working for a couple of versions now: #4311.

**Pull request merged**